tests/apis/avm/utxos.test.ts

```diff
@@ -79,14 +79,12 @@ let setMergeTester = (input:UTXOSet, equal:Array<UTXOSet>, notEqual:Array<UTXOSe
     let instr:string = JSON.stringify(input.getUTXOIDs().sort());
     for(let i:number = 0; i < equal.length; i++){
         if(JSON.stringify(equal[i].getUTXOIDs().sort()) != instr){
-            console.log("equal failed: ", input.getUTXOIDs().sort(), i, equal[i].getUTXOIDs().sort());
```

Comment:
Just curious why this log was removed. I guess it's fine to remove it.

Reply:
Just cleaning up.